.ado/pipelines/templates/jobs-ui-app-build.yaml

```diff
@@ -8,6 +8,11 @@ jobs:
   displayName: 'Build UI app'
   steps:
 
+  - task: NodeTool@0
+    displayName: 'Use Node v16 for UI app'
+    inputs:
+      versionSpec: '16.x'
```

Comment:
Especially given the fact that the version number is defined twice?

Reply:
great idea

Reply:
done